_source/logzio_collections/_log-sources/windows.md

```diff
@@ -145,6 +145,11 @@ LogFile %ROOT%\\data\\nxlog.log
 </Extension>
 ```
 
+See the
+[NXLog Documentation](https://nxlog.co/documentation/nxlog-user-guide/parsing-multiline.html#parsing-multiline)
+for more information on configuring multiline parsing of NXLog logs.
```

Comment:
We're parsing Windows logs here, right? If you don't specifically mean logs generated by NXLog, I'd suggest revising to something like this:

```suggestion
for more information on parsing multiline logs.
```

Reply:
I don't have any objections.